src/components/Data/Documents/Views/TimeSeries.vue

```diff
@@ -47,7 +45,7 @@
             :new-value="newCustomNumberField || ''"
             @update-color="updateColor"
             @autocomplete::change="
-              item => {
+              (item) => {
```

Comment:
Nitpicking: why did you add the parenthesis? We only need them with at least 2 variables

Reply:
I think it's an extension of my editor, I'll see if I can fix it.